### PR TITLE
use lexertl14

### LINKED ISSUE #224: The lexertl is an external fork; can the original be used?
In:
https://github.com/BlueBrain/MorphIO/tree/master/3rdparty

### PR DESCRIPTION
**Pull request by mgeplf**
* requires moving to c++14, should be safe
* fixes #224

### CODE REVIEW

**Comment by mgeplf** [comment]
This depends on the outcome of https://github.com/BlueBrain/MorphIO/issues/133

File: .gitmodules [hunk 1]
```diff
@@ -10,3 +10,6 @@
 [submodule "3rdparty/GSL_LITE"]
 	path = 3rdparty/GSL_LITE
 	url = https://github.com/martinmoene/gsl-lite.git
+[submodule "3rdparty/lexertl14"]
```
Comment (matz-e) [inline .gitmodules:13]: F'in Github won't allow me to comment above, but you still have `3rdparty/lexertl` in this file.

Comment (mgeplf) [reply .gitmodules:13]: I'll remove it, but it shouldn't matter (submodules are less than ideal)

**Comment by mgeplf** [comment]
Tested against NeuroM d177237532c62d627084a736bbda6a47b02015d4 as well.

**Review by matz-e — APPROVED** [review]

**Pull request merged**